trunk/Admin/js/src/partials/Actions.js

```diff
@@ -3,7 +3,7 @@ let AdminQuickbarActions = {
   /**
```

Comment:
Would it be usefull to separate the actions into their concerns? 

like one object swift inside the actions for all swift performance actions?